src/Core/Migration/V6_7/Migration1765287397AddConsentTable.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1765287397AddConsentTable extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1765287397;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeStatement('
+            CREATE TABLE IF NOT EXISTS `consent_state` (
+                `id` BINARY(16) NOT NULL,
+                `name` VARCHAR(100) NOT NULL,
+                `identifier` BINARY(16) NULL,
+                `state` VARCHAR(20) NOT NULL,
+                `actor_id` BINARY(16) NOT NULL,
+                `created_at` DATETIME(3) NOT NULL,
+                `updated_at` DATETIME(3) NULL,
```

Comment:
does it make sense to add a explicit `state_active_since` date field (naming is clearly not perfect), because it is quite hard now to find out since when some consent is in that state and i figure that is a important information to have?

We can rely on updated_at field, but that is automatically managed by the DAL which might lead to wrong results where update at is updated but the state not changed? 

Alternativately we could linkt to the latest consent log and rely on the created_at date of that to figure out since when that state is active?